### PR TITLE
Feature new pack

### PR DESCRIPTION
**Pull request by dianam21**
made a snapshot of tidyverse. please approve merge.

### CODE REVIEW

File: code/packages.R [hunk 1]
```diff
@@ -1 +1,2 @@
-
+install.packages("tidyverse")
+library(tidyverse)
```
Comment (terrymclaughlin) [inline code/packages.R]:
```suggestion
#library(tidyverse)
renv::snapshot()
```

File: code/packages.R [hunk 2]
```diff
@@ -1 +1,3 @@
-
+install.packages("tidyverse")
```
Comment (dianam21) [inline code/packages.R]:
```suggestion
#install.packages("tidyverse")
```

File: code/packages.R [hunk 3]
```diff
@@ -1 +1,3 @@
...
+#library(tidyverse)
```
Comment (dianam21) [inline code/packages.R]:
```suggestion
library(tidyverse)
```

File: code/packages.R [hunk 4]
```diff
@@ -1 +1,3 @@
...
+renv::snapshot()
```
Comment (dianam21) [inline code/packages.R]:
```suggestion
#renv::snapshot()
```

**Comment by terrymclaughlin** [comment]
Can't merge this into master.  Select develop instead.

**Pull request closed without merging**